atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -955,13 +972,23 @@ private void renderNamedDeleteRanges() {
             } line("}");
         }
 
+        private void renderOptimizeColumnSelection() {
+            line("private ColumnSelection optimizeColumnSelection(ColumnSelection columns) {"); {
+                line("if (columns.allColumnsSelected()) {"); {
+                    line("return allColumns;");
+                    line("}");
+                } line("return columns;");
+            } line("}");
+        }
```

Comment:
While the code here makes sense, I think it's not normally how the rendering code is structured. Something like the following is more common:

```
            line("private ColumnSelection optimizeColumnSelection(ColumnSelection columns) {"); {
                line("if (columns.allColumnsSelected()) {"); {
                    line("return allColumns;");
                } line("}");
              line("return columns;");
            } line("}");
```